src/commands/helm-lint.yaml

```diff
@@ -1,5 +1,9 @@
 ---
+parameters:
+  charts:
+    description: "Comma seperated list of charts to lint (relative path)"
+    type: "string"
 description: Lint Helm charts
 steps:
   - run: |
-      ct lint --validate-maintainers=false --check-version-increment=false --chart-dirs helm
+      ct lint --validate-maintainers=false --charts <<parameters.charts>>
```

Comment:
```suggestion
      ct lint --validate-maintainers=false --charts="helm/<<parameters.charts>>"
```

Reply:
This will hard fix to a single chart. I would like the command to be able to support multiple charts. Would make reuse of the command better.

Not sure there is much value is forcing only a single chart, when supporting multiple charts is trivial.

Reply:
> Let's just pass a single chart because the command has a **way** better output when single chart is passed.

The value 🔼 

The output for linting schema is useless when specifying multiple charts. Also I wouldn't worry about re-usability here. It's a command not a job.

Reply:
I checked that and I'm wrong. Apparently the output is different when `--charts-dir` is specified. Okay let's keep it if it's important to you.